### PR TITLE
Refactor routes

### PR DESCRIPTION
**Pull request by andygw**
So much cleaner...

image

### CODE REVIEW

**Comment by andygw** [comment]
Important note: seed account PWs changed to `111111`

**Comment by squush** [comment]
Sounds good. Why the change?

**Comment by andygw** [comment]
Easier to type, I guess? I can change it back if you have the pws saved already

**Comment by andygw** [comment]
Changed the seed to have some consistent users, crimes, and bookings each time. The My Bookings page will automatically look like this after each re-seed:

image

There will also be some random users and crimes added after this, to seed the All Crimes page a bit.

Also added a couple to-do notes.

(This should have been on a different branch but I screwed up)

File: app/views/bookings/_form.html.erb [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <%= simple_form_for [@crime, @booking] do |f| %>
   <%= f.input :target %>
+  <%# change to calendar / look up 'flatpciker' %>
```
Comment (Sreg1) [inline app/views/bookings/_form.html.erb:3]: did you mean "flatpickr"

**Review by Sreg1 — APPROVED** [review]

**Pull request merged**